docker-compose-quick.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.4'
+
+services:
+
+  lpvs:
+    build:
+      context: .
+    image: ghcr.io/samsung/lpvs:latest
+    restart: always
+    ports:
+      - "7896:7896"
+    environment:
+      ## Required if frontend and backend are different
+      ## Database Configuration
+      - spring.datasource.url=jdbc:mysql://mysqldb:3306/lpvs
+      - spring.datasource.username=root
+      - spring.datasource.password=
+      - spring.jpa.properties.hibernate.default_schema=lpvs
+      ## Github data for fetching code
+      - github.login=
+      - github.token=
+      - github.api.url=https://api.github.com
+      - github.secret=LPVS
+      ## Google OAuth Login
+      - spring.security.oauth2.client.registration.google.client-id=GOOGLE_CLIENT_ID
+      - spring.security.oauth2.client.registration.google.client-secret=GOOGLE_CLIENT_SECRET
+      - spring.security.oauth2.client.registration.google.redirect-uri=http://localhost:7896/login/oauth2/code/google
+      - spring.security.oauth2.client.registration.google.scope=profile, email
+      ## Github OAuth Login
+      - spring.security.oauth2.client.registration.github.client-id=GITHUB_CLIENT_ID
+      - spring.security.oauth2.client.registration.github.client-secret=GITHUB_CLIENT_SECRET
+      - spring.security.oauth2.client.registration.github.redirect-uri=http://localhost:7896/login/oauth2/code/github
+      - spring.security.oauth2.client.registration.github.scope=user
```

Comment:
Do we really need this for the default profile? These options are used in Web profile.

Reply:
I've tested solution without this part and we got error like below:
`Exception in thread "main" java.lang.ClassNotFoundException: org.springframework.boot.loader.thin.ThinJarLauncher
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at org.springframework.boot.loader.wrapper.ThinJarWrapper.launch(ThinJarWrapper.java:138)
	at org.springframework.boot.loader.wrapper.ThinJarWrapper.main(ThinJarWrapper.java:107)
`

I am not sure how to remove this part properly, so I tend to keep it